grunt.js

```diff
@@ -314,6 +287,18 @@ module.exports = function(grunt) {
         src: ['build/temp/video.js', 'node_modules/vtt.js/dist/vtt.js'],
         dest: 'build/temp/video.js',
       },
+    },
+    shell: {
+      // Run multiple watch tasks in parallel
+      // Needed so watchify can cache intelligently
+      watchAll: {
+        command: [
+          'grunt connect:dev',
+          'grunt watch',
+          'grunt browserify:watch',
+          'grunt karma:watch'
+        ].join('&')
+      }
```

Comment:
Is there a benefit of doing it this way over something like [grunt-concurrent](https://github.com/sindresorhus/grunt-concurrent)?